frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -205,7 +208,7 @@ export default function EditorDialog(props: EditorDialogProps) {
       <Box paddingTop={2} height="100%">
         <Editor
           language="yaml"
-          theme="vs-dark"
+          theme={themeName === 'dark' ? 'vs-dark' : 'light'}
```

Comment:
The reason why we hardcoded the dark one was that Monaco Ed was not updating/behaving correctly when we switched the theme. Please double-check this before we merge this change in.

Reply:
It updates the theme and works correctly now. I guess the last monaco dependency update must have fixed it.